src/compiler/types.ts

```diff
@@ -1212,11 +1212,11 @@ namespace ts {
          */
         emit(targetSourceFile?: SourceFile, writeFile?: WriteFileCallback): EmitResult;
 
-        getSyntacticDiagnostics(sourceFile?: SourceFile): Diagnostic[];
+        getOptionsDiagnostics(): Diagnostic[];

```

Comment:
`getOptionsDiagnostics` sounds vague enough that it's not obvious that it refers to compiler options. Can we not just keep that name?